presto-router/src/main/resources/router_ui/index.html

```diff
@@ -27,6 +27,12 @@
     <!-- Sparkline -->
     <script type="text/javascript" src="vendor/jquery.sparkline/jquery.sparkline.min.js"></script>
 
+    <script type="text/javascript" >
```

Comment:
This inline script won't cause any issues until CSP is enabled on the presto-router. I just had another PR to fix the CSP errors and found the same inline script. I moved it directly into the index.jsx. So it will be in the index.js directly.

You don't have to fix it now. Just share the info with you if the CSP header needs to be enabled in the presto-router in the future.